modules/claims_api/app/services/claims_api/evidence_waiver_document_service.rb

```diff
@@ -25,11 +26,23 @@ def generate_body(claim:, doc_type:, pdf_path:, ptcpnt_vet_id:)
       auth_headers = claim.auth_headers
       veteran_name = compact_veteran_name(auth_headers['va_eauth_firstName'],
                                           auth_headers['va_eauth_lastName'])
-      file_name = build_file_name(veteran_name:, identifier: claim.claim_id, suffix: '5103')
+      file_name = built_file_name(veteran_name:, claim:, suffix: FORM_SUFFIX)
       tracked_item_ids = claim.tracked_items&.map(&:to_i) if claim&.has_attribute?(:tracked_items)
 
       generate_upload_body(claim_id: claim.claim_id, system_name: 'VA.gov', doc_type:, pdf_path:, file_name:,
                            birls_file_number: nil, participant_id: ptcpnt_vet_id, tracked_item_ids:)
     end
+
+    def built_file_name(veteran_name:, claim:, suffix:)
```

Comment:
There already s a `build_file_name` method in the parent class so I was looking for something to differentiate that this was returning the _built_ name.  But either way `build_file_name` is not available, totally open to other suggestions.

Reply:
Could we override the parentʼs `build_file_name` if we have a more specific use-case here? Slightly renaming a method that does the same thing to avoid conflict seems confusing, imo.

Reply:
I believe this is related to the above comment as well, _probably_ covered by that too.

Reply:
Yup, we would only need to override it here if we were changing the behavior somehow. But we should be covered by the base class method.